ra/ra.go

```diff
@@ -1960,18 +1960,25 @@ func (ra *RegistrationAuthorityImpl) DeactivateRegistration(ctx context.Context,
 }
 
 // DeactivateAuthorization deactivates a currently valid authorization
-func (ra *RegistrationAuthorityImpl) DeactivateAuthorization(ctx context.Context, auth core.Authorization) error {
-	if auth.Status != core.StatusValid && auth.Status != core.StatusPending {
-		return berrors.MalformedError("only valid and pending authorizations can be deactivated")
+func (ra *RegistrationAuthorityImpl) DeactivateAuthorization(ctx context.Context, req *corepb.Authorization) (*emptypb.Empty, error) {
+	if req == nil || req.Id == "" || req.Identifier == "" || req.Status == "" || req.Expires == 0 {
```

Comment:
The only values we actually need to check here are `req.Id` and `req.Status`; the others aren't relevant for Deactivation.